main/samba/src/scripts/user-quota

```diff
@@ -30,8 +30,9 @@
 # uidNumber: user uidNumber
 # quota: quota size in KB
 use Quota;
-
-sub currentQuota
```

Comment:
By the way in this method, we should do the same and check the other new mount_points to return the actual quota right?